packages/server/src/fhir/operations/patienteverything.ts

```diff
@@ -80,5 +80,9 @@ export async function getPatientEverything(
     filters,
     count: params?._count ?? defaultMaxResults,
     offset: params?._offset,
+    include: [{ resourceType: 'Patient', searchParam: 'organization' }],
+    revInclude: [
+      { resourceType: 'Location', searchParam: 'organization', targetType: 'Organization', modifier: Operator.ITERATE },
```

Comment:
If not too annoying, can we add a test for `Organization` and/or `Location`?

I'm not concerned about this not working.  I'm thinking of it more as a matter of capturing the requirement when future engineers inevitably touch this code.